File: app/controllers/Application.java
package controllers;

import play.mvc.Controller;
import play.mvc.Result;
import constants.AppConstants;

import javax.inject.Inject;

public class Application extends Controller {

    @Inject
    AppConstants ca;

    public Result index() {
        return ok(views.html.index.render(ca.AuthorizationUrl()));
    }

}

File: app/controllers/Application.scala
package controllers

import javax.inject.Inject

import constants.AppConstants
import play.api.mvc.{Result, Action, Controller}


class Application @Inject() (ac: AppConstants) extends Controller{
  import ac.{AuthorizationUrl, Scopes, IndexPage}
  import Scopes._

  def index = Action {
    Ok(IndexPage(AuthorizationUrl, Likes, Comments, Relationships))
  }

}
